stdlib/Sockets/src/Sockets.jl

```diff
@@ -471,6 +471,31 @@ function connect(sock::LibuvStream, args...)
     return sock
 end
 
+"""
+    nagle(socket::Union{TCPServer, TCPSocket}, enable::Bool)
+
+Enables or disables Nagle's algorithm on a given TCP server or socket.
+"""
+function nagle(sock::Union{TCPServer, TCPSocket}, enable::Bool)
+    # disable or enable Nagle's algorithm on all OSes
+    ccall(:uv_tcp_nodelay, Cint, (Ptr{Cvoid}, Cint), sock.handle, Cint(!enable))
```

Comment:
Error checking of return value?

Reply:
Good idea, I'll make a separate PR for this.